README.md

```diff
@@ -14,18 +14,18 @@ moveit)
 
     git clone https://github.com/moveit/moveit_drake.git
     cd moveit_drake
-    docker-compose -f .docker/docker-compose.yaml build
+    docker compose build
 
 This should give you an image with `drake` and `moveit2`. Next, create a
 container with the following and create shell access.
 
-    docker-compose -f .docker/docker-compose.yaml up
-    docker exec -it moveit_drake bash
```

Comment:
This still needs to be moveit_drake as the container will be named moveit_drake from the docker-compose file.

Reply:
Shall I just make the necessary change after mergint it?

Reply:
Yeah sorry, should be `docker compose exec -it moveit bash`. Feel free to merge and fix yourself!

Reply:
Wait, just re-read this. My changes work as well, should be fine